docs/linux.md

```diff
@@ -17,6 +17,8 @@ sudo chmod +x /usr/bin/ollama
 
 ## Start Ollama
 
+NOTE : If you are trying to run Ollama after fresh installation then you can skip to step running model (Since Ollama server should already be running at port number 11434 after a fresh installation)
```

Comment:
This guide is meant as a manual install guide (vs the automated install script). We don't want to conflate the two here.

Reply:
The issue is solved by @mxyng 👍